### DISCUSSION TITLE
TuyaMCU command to set time

### DISCUSSION THREAD

**Discussion by belveder79** [arendst/Tasmota#16202] [Q&A] [answered]
I have flashed [this](https://templates.blakadder.com/digoo_DG-ZXGS21.html) gas sensor and it works pretty fine. The only annoying thing is that the time on the display (controlled by the TuyaMCU) is wrong. 

I'm at UTC+01:00 with daylight saving right now at +01:00 currently, eg. UTC being at 9:00 and me being at 11:00. The time it shows at 11:00 is 10:00 which is just off by -01:00, but I can't really make sense out of it. I believe it ignores daylight saving and just shows UTC and that's it. 

And here's the question. Does anyone have an idea how to adjust the time on the TuyaMCU, respectively to "rule" or "command" some offset to the MCU to use the correct time?

**Comment by sfromis** [comment, marked as answer]
Did you configure Tasmota to match your time zone? If not, try that, to have your correct local time visible in the console and so on.

I do not know much about TuyaMCU operation, but to me it appears like Tasmota would forward it's time to the TuyaMCU device, and since you even have a sensible time at all, it would appear that this is happening.

**Comment by belveder79** [comment]
Thx... will try asap!

**Comment by belveder79** [comment]
works, thx
